language/examples/bqml/bqml_llm_examples.ipynb

```diff
@@ -0,0 +1,944 @@
+{
```

Comment:
<p>The word "create" in "create the LLM model" might be confusing as it sounds like the user is creating/training their own LLM model, which they aren't -- instead they're creating a BigQuery ML model that is a remote connection to the Vertex AI LLM. Perhaps it would be better to rephrase this to be clearer.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>good shout thanks, renamed to configure to be more consistent with documentation/blogs</p>

Reply:
<p>Ah, "executing LLMs" sounds a bit strange. I think it might be better to say "Using LLMs with BigQuery ML" and "To use LLMs in BQML, ..."</p>